openadapt/app/__init__.py

```diff
@@ -1,13 +0,0 @@
-"""openadapt.app package.
-
-This package contains modules for the OpenAdapt application.
-
-Example usage:
-    from openadapt.app import run_app
-
-    run_app()
-"""
-from openadapt.app.main import run_app
-
-if __name__ == "__main__":
```

Comment:
Is there a reason why you deleted this? It was like this before my PR was merged
```python
from openadapt.app.main import run_app

if __name__ == "__main__":
    run_app()
```

Reply:
it was causing some issues, and isn't really needed. It was removed a while ago I think

Reply:
@0dm Hey Aaron, can you run ```black --check .```  to see if any files need reformatting and ```flake8 .```  to see if any lint errors come up. If those pass then the PR should be ready for merge.

Reply:
yep, there was a few things missing -- should be good now